R/adaptive.bin.R

```diff
@@ -144,12 +137,7 @@ adaptive.bin <- function(x,
         that <- this_table |> dplyr::filter(mz > boundaries$lower & mz <= boundaries$upper)
 
         if (nrow(that) > 0) {
-          that <- combine.seq.3(that)
-
-          if (nrow(that) != 1) {
-            that <- that[order(that[, 1]), ]
-          }
-
+          that <- combine.seq.3(that) |> dplyr::arrange_at("mz")
```

Comment:
I think idk maybe we should try to find a balance between using dplyr function and base function especially when using base function doesn't make code ugly and less readable because dplyr function are quite slow.